app/editor/src/features/content/papers/Papers.tsx

```diff
@@ -167,26 +139,33 @@ const Papers: React.FC<IPapersProps> = (props) => {
     async (filter: IContentListFilter & Partial<IContentListAdvancedFilter>) => {
       try {
         setIsLoading(true);
-        const results = await findContentWithElasticsearch(toFilter(filter), true);
+        const results = await findContentWithElasticsearch(
+          toFilter({
+            ...filter,
+          }),
+          true,
+        );
         const items = results.hits.hits
           .filter((h) => !!h._source)
-          .map((h) => castContentToSearchResult(h._source! as IContentModel));
+          .map((h) => h._source! as IContentModel)
+          .sort(sortContent(filter.sort))
```

Comment:
We now apply the sort after in the UI too, so that we handle when Elasticsearch doesn't have the latest source.sortOrder.